test/CMakeLists.txt

```diff
@@ -6,12 +6,16 @@ function(set_test_env _testname)
   set_property(TEST ${_testname} APPEND PROPERTY ENVIRONMENT
       LD_LIBRARY_PATH=$<TARGET_FILE_DIR:edm4hep>:$<TARGET_FILE_DIR:podio::podio>:$<$<TARGET_EXISTS:edm4hepRDF>:$<TARGET_FILE_DIR:edm4hepRDF>:>$<TARGET_FILE_DIR:ROOT::Core>:$ENV{LD_LIBRARY_PATH}
       ROOT_INCLUDE_PATH=${PROJECT_SOURCE_DIR}/edm4hep:${PROJECT_SOURCE_DIR}/utils/include:$ENV{ROOT_INCLUDE_PATH}
+      PYTHONPATH=${PROJECT_SOURCE_DIR}/python:$ENV{PYTHONPATH}
```

Comment:
We either need to add the podio that we have used for building here as well to `PYTHONPATH`, or we have to do it in the workflow file in the step where we build and install podio. CI is partially failing, because it picks up an older version of podio from the LCG stack:

https://github.com/key4hep/EDM4hep/actions/runs/10045234420/job/27762066625?pr=272#step:4:817